students/tim_lurvey/lesson01/Activity/test_unit.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+import sys
+import unittest
+
+sys.path.append(r"C:\\Users\\pants\\PycharmProjects\\SP_Python220B_2019\\")
+# from lessons.lesson01.activity.calculator import calculator
+from lessons.lesson01.activity.calculator.adder import Adder
+from lessons.lesson01.activity.calculator.subtracter import Subtracter
+from lessons.lesson01.activity.calculator.multiplier import Multiplier
+from lessons.lesson01.activity.calculator.divider import Divider
+from Activity.calc.calc_fix import Calculator
+
+
+# class BaseTest(unittest.TestCase):
```

Comment:
Always remove commented out code. It's ok to comment out something for a couple of hours while you do debug. If you leave it there, the moment your release it, someone else will uncomment it out of ignorance or pure evil and break your code.